db-scripts/src/main/resources/cgds.sql

```diff
@@ -775,6 +791,22 @@ CREATE TABLE `clinical_event_data` (
 );
 
 -- --------------------------------------------------------
+DROP TABLE IF EXISTS `reference_genome_gene`;
+CREATE TABLE `reference_genome_gene` (
+    `ENTREZ_GENE_ID` int(11) NOT NULL,
+    `CYTOBAND` varchar(64) NOT NULL,
+    `EXONIC_LENGTH` int(11) DEFAULT NULL,
+    `GENE_START` bigint(20) DEFAULT NULL,
+    `GENE_END` bigint(20) DEFAULT NULL,
+    `GENE_STABLE_ID` varchar(64) DEFAULT NULL,
```

Comment:
I am not quite sure what we want to call HUGO_GENE_SYMBOL_START or HUGO_GENE_START?

Reply:
GENE_STABLE_ID is Ensembl ID something like ENSG00000121410, not HUGO_GENE_SYMBOL in gene table. Trevor suggested to have it because widely used easy to search.

Reply:
Suggest we call it ENSEMBL_GENE_ID (like we have fields called ENTREZ_GENE_ID) for consistency and to reduce ambiguity.